tiledb/bioimg/converters/base.py

```diff
@@ -37,49 +22,34 @@ def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
 
     @property
     @abstractmethod
-    def level_count(self) -> int:
-        """Return the number of levels for this multi-resolution image.
-        Levels are numbered from 0 (highest resolution) to level_count - 1 (lowest resolution).
-
-        :return: The number of levels in the slide
-        """
+    def axes(self) -> Axes:
+        """The axes of this multi-resolution image."""
 
+    @property
     @abstractmethod
-    def level_axes(self, level: int) -> Axes:
-        """Return the axes for the given level.
-
-        :param level: Number corresponding to a level
+    def level_count(self) -> int:
```

Comment:
Have you tried building the html page with the updated docstrings? All the derived classes get the `docstrings` from `base` class `docstrings`.  `Docstrings` follow `reStructuredText` format do we want to change that?

Reply:
Just tried it, looks good to me. It can still be parsed as reStructuredText, I just removed the duplicated `:param:`/`:return:` text that's already in the method description. 
![Screenshot from 2022-11-21 20-07-26](https://user-images.githubusercontent.com/291289/203128868-1864f5eb-034b-4c02-b9cc-debc2f87dc86.png)

Reply:
It would render for sure. I get that these are `small` methods or properties and that `params` and `returns` feel verbose, but it creates this heterogeneity between `write_level_image` and `write_group_metadata` for example where in the `metadata`  receive explanation in the former but not in the latter. And secondly imo each API should be self-contained and self-explanatory without the need to look somewhere else in the docs. Anyway your call at this dev stage probably it does not make any difference.

Reply:
Ok I see what you mean, indeed this trades off the redundancy in docstrings with occasional inconsistency; some parameters are self-explanatory, some aren't (though in the example you mentioned i'd rather remove the verbosity  in `write_level_image` than bring it back for all the rest).

As for the second point, what is not self-contained/self-explanatory? There's no linking to docs elsewhere afaict.

Reply:
I mean easy indexing as a user if i was about to look up at a function signature and only that I would like to see all arguments explained in the docstrings of the function without the need to look up somewhere else to find out what does metadata means, or which metadata we refer to for example. e.g Imagine `numpy` [transpose function](https://numpy.org/doc/stable/reference/generated/numpy.transpose.html) everybody knows what `axes` knows and it is used in many functions around the package. However as I said at this stage of dev might not make any sense and this might also be my personal preference so your call :)

Reply:
Sure, no disagreement about that. I'm only arguing against the redundancy within the same docstring that doesn't add any new information:
```
def f(x,y):
  """
  Returns  {blahblah} given {the Foo x} and {the bar y}.

 :param x: {the Foo x}
 :param y: {the Bar y}
  
  :return: {blahblah}
  """
````